vm-deployment/vm.bicep

```
@@ -0,0 +1,282 @@
+/*
+Virtual machine deployment.
+*/
+
+// Parameters for resource names
+param virtualMachines string = 'iagora'
+param publicIPAddresses string = 'iagora-ip'
+param virtualNetworks string = 'iagora-vnet'
+param networkInterfaces string = 'iagora731_z1'
+param networkSecurityGroups string = 'iagora-nsg'
+
+// SSH public key parameter
+@description('The SSH public key to use for authentication.')
```

```suggestion
@description('The SSH public key to use for authentication.')
```
Doesn't the description replace the comment? Same suggestion for other comment-next-to-description occurrences.